src/fog/Fog.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+import 'wicg-inert'
+
+import { StyledFog, StyledFogContainer } from './Fog.style'
+
+export type FogProps = Readonly<{
+  isLoading?: boolean
+  children: React.ReactNode
+}>
+
+export const Fog = ({ isLoading, children }: FogProps) => (
+  <StyledFogContainer>
+    <StyledFog $isLoading={isLoading} aria-hidden="true" />
+    <div ref={node => node && isLoading && node.setAttribute('inert', '')} aria-hidden={isLoading}>
```

Comment:
Don't we want to remove the `inert` attribute when `isLoading` is set back to `false`?

Reply:
By looking at the devTools it does, but I'm wondering if it's the way storybook renders the component. Maybe I should add a else with an explicite `removeAttribute`

Reply:
Maybe it's because React set all the attributes during render (remove old ones) and then the ref callback is called.

Reply:
Done 👍